test_runner/src/main/kotlin/ftl/run/platform/android/ResolveApks.kt

```diff
@@ -3,16 +3,18 @@ package ftl.run.platform.android
 import ftl.args.AndroidArgs
 import ftl.args.yml.ResolvedApks
 
-internal fun AndroidArgs.resolveApks() = listOf(
-    element = ResolvedApks(
-        app = appApk,
-        test = testApk,
-        additionalApks = additionalApks
-    )
+internal fun AndroidArgs.resolveApks() = listOfNotNull(
+    element = appApk?.let { appApk ->
+        ResolvedApks(
+            app = appApk,
+            test = testApk,
+            additionalApks = additionalApks
+        )
+    }
 ).plus(
     elements = additionalAppTestApks.map {
         ResolvedApks(
-            app = it.app ?: appApk,
+            app = it.app ?: appApk!!,
```

Comment:
I think should throw here custom exception, not nasty NPE (which should not happen in kotlin app)

Reply:
agreed, I don't like force unwraps in the production code.